From bunk@stusta.de Mon Nov 21 13:09:07 2005
Date: Sun, 20 Nov 2005 23:56:11 +0100
From: Adrian Bunk <bunk@stusta.de>
To: <gregkh@suse.de>
Cc: <mdharm-usb@one-eyed-alien.net>
Subject: USB: small cleanups
Message-ID: <20051120225611.GA16060@stusta.de>
Content-Disposition: inline

This patch contains the following cleanups:
- make needlessly global functions static
- every file should #include the headers containing the prototypes for
  it's global functions


Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/atm/usbatm.c   |    4 ++--
 drivers/usb/serial/io_ti.c |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

--- gregkh-2.6.orig/drivers/usb/atm/usbatm.c
+++ gregkh-2.6/drivers/usb/atm/usbatm.c
@@ -646,14 +646,14 @@ static void usbatm_destroy_instance(stru
 	kfree(instance);
 }
 
-void usbatm_get_instance(struct usbatm_data *instance)
+static void usbatm_get_instance(struct usbatm_data *instance)
 {
 	dbg("%s", __func__);
 
 	kref_get(&instance->refcount);
 }
 
-void usbatm_put_instance(struct usbatm_data *instance)
+static void usbatm_put_instance(struct usbatm_data *instance)
 {
 	dbg("%s", __func__);
 
--- gregkh-2.6.orig/drivers/usb/serial/io_ti.c
+++ gregkh-2.6/drivers/usb/serial/io_ti.c
@@ -2843,7 +2843,7 @@ static struct edge_buf *edge_buf_alloc(u
  * Free the buffer and all associated memory.
  */
 
-void edge_buf_free(struct edge_buf *eb)
+static void edge_buf_free(struct edge_buf *eb)
 {
 	if (eb) {
 		kfree(eb->buf_buf);
